### PR TITLE
Find maximum value

### PR DESCRIPTION
**Pull request by shaybrow**
- [ ] Top-level README “Table of Contents” is updated
 - [x] Feature tasks for this challenge are completed
 - [x] Unit tests written and passing
     - [x] “Happy Path” - Expected outcome
     - [x] Expected failure
     - [x] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [x] Picture of whiteboard

### CODE REVIEW

File: challenges/lib/src/main/java/challenges/tree/BinaryTree.java [hunk 1]
```diff
@@ -62,9 +83,23 @@ public void _inOrder (Node input, ArrayList<Integer> inputList ) {
...
+        if (this.root == null) throw new Exception("Can't find max on null tree");
+
+        this._findMaxValue(this.root);
+        return maxValue;
+    }
+    public void _findMaxValue (Node input) {
+        if (input == null) return ;
+//        if (maxValue == null) maxValue = input.val;
+        if (maxValue < input.val) maxValue = input.val;
+        this._findMaxValue(input.left);
```
Comment (Mattpet26) [inline challenges/lib/src/main/java/challenges/tree/BinaryTree.java:101]: Great work with the recursion!
